src/App.tsx

```diff
@@ -1,14 +1,42 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+import { Sort } from './types/Sort';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoaded, setIsLoaded] = useState(true);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [userId, setUserId] = useState(0);
+  const [query, setQuery] = useState('');
+  const [statusFilter, setStatusFilter] = useState<Sort>(Sort.all);
+
+  useEffect(() => {
+    getTodos().then(todosFromServer => {
+      setTodos(todosFromServer);
+      setIsLoaded(false);
+    });
+  }, []);
+
+  const filteredTodos = todos.filter(todo => {
+    switch (statusFilter) {
+      case Sort.completed:
+        return todo.completed && todo.title.toLowerCase().includes(query.toLowerCase());
+      case Sort.active:
+        return !todo.completed && todo.title.toLowerCase().includes(query.toLowerCase());
+      default:
+        return todo.title.toLowerCase().includes(query.toLowerCase());
+    }
```

Comment:
Let's create a var for `todo.title.toLowerCase().includes(query.toLowerCase())` expression

Reply:
Cause you use it 3 times